srcs/utils/memory_utils.c

```diff
@@ -0,0 +1,20 @@
+#include "minishell.h"
+
+void	free_minishell(t_minishell *minishell)
+{
+	printf("Theoretically, I freed minishell.\n");
+	(void)minishell;

```

Comment:
What? 😂 Where are you freeing it?

Reply:
But why later, don't think I like to leave this half done, if I see a function called free_minishell I'll assume it is actually freeing it

Reply:
Well, as soon as we agree on some sort of beginning of structure for Minishell, and one part of it will have assigned memory, I'll do it. But I can't free something that doesn't exist yet and has no memory assigned ^^

Reply:
Unless you have an idea 😂

Reply:
Ah ok, I see your point